### ISSUE TITLE
Get an error when I try to Highlight.

### ISSUE THREAD

**Issue by aweall** [Neftas/supplementary-buttons-anki#31]
```
An error occurred. It may have been caused by a harmless bug, 
or your deck may have a problem. 
To confirm it's not a problem with your deck, please run Tools > Check Database. 
If that doesn't fix the problem, please copy the following
into a bug report:
Traceback (most recent call last):
  File "aqt/webview.pyc", line 18, in run
  File "aqt/editor.pyc", line 449, in bridge
  File "aqt/editor.pyc", line 497, in mungeHTML
  File "aqt/editor.pyc", line 870, in _filterHTML
  File "BeautifulSoup.pyc", line 612, in __getitem__
KeyError: 'style'
```
running anki-2.0.32  on mac os x 10.10.2

**Comment by Neftas** [comment]
Thanks for taking the time to report this behavior.

From the looks of things it seems that Anki strips the `style` attribute from the `font` tags, thus removing the highlighting and giving this error. This only happens on Windows and Mac machines, so on Linux everything works fine. [Related bug](https://github.com/Neftas/supplementary-buttons-anki/issues/11)

**Comment by Neftas** [comment]
Disabled text highlighting 43286080748cde7d7a09f9bb47837a78c8bb4097 on all platforms except Linux, where it works like it should. Anki strips out highlighting from the HTML on Mac OS and Windows.

**Comment by Neftas** [comment]
Fixed in faa9e2c672fa8177d2968569df9dab32e5b3a8a9.

Please re-install the addon.
